tests/scripts/arm32_ci_script.sh

```diff
@@ -102,7 +102,15 @@ function check_git_head {
 function unmount_rootfs {
     local rootfsFolder="$1"
 
-    if grep -qs "$rootfsFolder" /proc/mounts; then
+    #Check if there are any open files in this directory.
+    if [ -d $rootfsFolder ]; then
+        #If we find information about the file
+        if [ sudo lsof +D $rootfsFolder ]; then
```

Comment:
The square brackets are not needed around `sudo lsof ...`. You can write that as `if sudo lsof ...; then`